docs/source/operations/transformations/horner.rst

```diff
@@ -133,7 +186,7 @@ describing real and complex polynomials can't be mixed.
 
 .. option:: +inv_origin=<northing,easting>
 
-    Coordinate of origin for the inverse mapping
+    Coordinate of origin for the inverse mapping. Not required for iterative inversion.
```

Comment:
Perhaps these are better placed under optional parameters with this change in effect? As far as I understand they are only necessary if you are explicitly defining an inverse operation. Same comment applies for `+inv_u` and `+inv_v`

Reply:
@kbevers  thanks for the review! For +inv_u, +inv_v, yes they are  optional and i will move them to the optional parameters. As for +inv_origin we still need it in the case of complex polynomials but it is not required in the case of real polynomials. How do you think we can handle this ?

Reply:
Your current wording for`+inv_origin`is probably okay then.